README.md

```diff
@@ -39,8 +39,396 @@ Then simply import:
+      filePath = resp.path();
+      return resp.readFile('base64');
+    })
+    .then(async base64Data => {
+      base64Data = `data:${type};base64,` + base64Data;
+      await Share.open({ url: base64Data });
+      // remove the image or pdf from device's storage
+      await RNFS.unlink(filePath);
+    });
+}
+```
+#### Static Values for Instagram Stories
+
+These can be assessed using Share.Social property
+For eg.
+```javascript
+import Share from 'react-native-share';
+
+const shareOptions = {
+    method: Share.InstagramStories.SHARE_BACKGROUND_AND_STICKER_IMAGE,
+    backgroundImage: 'http://urlto.png',
+    stickerImage: 'data:image/png;base64,<imageInBase64>', //or you can use "data:" link
+    backgroundBottomColor: '#fefefe',
+    backgroundTopColor: '#906df4',
+    attributionURL: 'http://deep-link-to-app', //in beta
+    social: Share.Social.INSTAGRAM_STORIES
+};
+Share.shareSingle(shareOptions);
+```
+
+Supported options for INSTAGRAM_STORIES:
+
+| Name  | Type     | Description |
+| :---- | :------: | :--- |
+| backgroundImage | string   | URL you want to share |
+| stickerImage | string   | URL you want to share |
+| method | string   | [List](#instagram-stories-method-list) |
+| backgroundBottomColor | string   |  (optional) default #837DF4 |
+| backgroundTopColor | string   | (optional) default #906df4 |
+| attributionURL | string   | (optional) facebook beta-test |
+| backgroundVideo | File   | URI you want to share |
+
+#### Instagram stories method list
+| Name  | Required options    |
+| :---- | :------: |
+| **SHARE_BACKGROUND_IMAGE** | backgroundImage   |
+| **SHARE_BACKGROUND_VIDEO** | backgroundVideo   |
+| **SHARE_STICKER_IMAGE** | stickerImage   |
+| **SHARE_BACKGROUND_AND_STICKER_IMAGE** | backgroundImage, stickerImage   |
+
+### Static Values for Facebook Stories
+These can be assessed using Share.Social property
+For eg.
+```javascript
+import Share from 'react-native-share';
+
+const shareOptions = {
+    method: Share.FacebookStories.SHARE_BACKGROUND_AND_STICKER_IMAGE,
+    backgroundImage: 'http://urlto.png', // url or an base64 string
+    stickerImage: 'data:image/png;base64,<imageInBase64>', //or you can use "data:" url
+    backgroundBottomColor: '#fefefe',
+    backgroundTopColor: '#906df4',
+    attributionURL: 'http://deep-link-to-app', //in beta
+    appId: '219376304', //facebook appId
+    social: Share.Social.FACEBOOK_STORIES
+};
+Share.shareSingle(shareOptions);
+```
+
+Supported options for FACEBOOK_STORIES:
+
+| Name  | Type     | Description |
+| :---- | :------: | :--- |
+| appId | string   | (required) facebook appId |
+| backgroundImage | string   | URL you want to share |
+| stickerImage | string   | URL you want to share |
+| method | string   | [List](#instagram-stories-method-list) |
+| backgroundBottomColor | string   |  (optional) default #837DF4 |
+| backgroundTopColor | string   | (optional) default #906df4 |
+| attributionURL | string   | (optional) facebook beta-test |
+
```

Comment:
We need to place this in the desired location at our docs

Reply:
@slaycky do you need help with that? If you have any doubt we are using docusarus to our docs. You can read more about that [here](https://react-native-community.github.io/react-native-share/docs/contributing)

Reply:
Thank you @MateusAndrade, I removed the lines, Can you check please if it's right?

Reply:
The README.MD is right now, but we still need to add these changes to the DOCS at: [share-open](https://react-native-community.github.io/react-native-share/docs/share-open) or [share-single](https://react-native-community.github.io/react-native-share/docs/share-single). Do you need help with that?

Reply:
Hello @MateusAndrade can ou help me? I have to change this file https://github.com/react-native-community/react-native-share/compare/master...slaycky:patch-2?

Reply:
Yes! This one: https://react-native-community.github.io/react-native-share/docs/share-single or https://github.com/react-native-community/react-native-share/edit/master/website/docs/share-single.mdx.